### DISCUSSION TITLE
Getting access to existing RSD or collections

### DISCUSSION THREAD

**Discussion by madhav85** [wgu-opensource/osmt#194] [Asking for Help] [answered]
I have recently gone through the OSMT Video and am wondering If there is any way to get access to pre-existing RSDs or collections?
In the video, Allison mentioned around 13K pre-existing RSDs. Are these available for us to use?

**Comment by JohnKallies** [comment, marked as answer]
You can find them here: https://www.wgu.edu/lp/skills-library.html

Please note the instructions regarding batch importing RSDs and mapping occupation codes: https://osmt.io/docs.html

OSMT doesn't have a "delete RSD" feature. From seeing what others have learned onboarding to OSMT, I recommend importing first in small batches and verifying that occupation/job codes are mapping as you expect them to map. Even better, do you imports on a throwaway instance of OSMT, like a Quickstart eval. Get the mappings right, and then import into your real OSMT.

I hope this helps.

John
